DataConnectors/M365Defender-VulnerabilityManagement/azureDeploy.json

```diff
@@ -319,211 +290,114 @@
               "name": "WEBSITE_RUN_FROM_PACKAGE",
               "value": "1"
```

Comment:
The value is set to 1, will it fetch zip from repo ?

Reply:
Correct.